packages/commercetools/theme/pages/Checkout/Billing.vue

```diff
@@ -226,7 +226,7 @@
             {{ $t('Continue to payment') }}
           </SfButton>
           <nuxt-link
-            to="/checkout/shipping"
+            :to="localePath('/checkout/shipping')"
```

Comment:
even better if there will be posibility to use object
```suggestion
            :to="localePath({name: 'shipping' })"
```